docker/Dockerfile

```diff
@@ -15,7 +15,7 @@ FROM centos:7
 LABEL maintainer="mozdef@mozilla.com"
 
 ENV NODE_VERSION 4.7.0
-ENV METEOR_VERSION 1.4.4.2
+ENV METEOR_VERSION 1.4.2.3
```

Comment:
@jeffbryner You recently modified this, however we aren't actually running the web ui using that version. https://github.com/mozilla/MozDef/blob/master/meteor/.meteor/release is the file that determines what version of meteor we will use (if we have a more recent version of meteor and try to build, we actually will download the version specified in that file).

I decided to revert it back to what we're running. If we choose to update meteor, we'll have to modify that file and then test things on qa as well.

Reply:
Does it break something to update?

Reply:
In order to update, we'd have to update node as well (in production too). Since this task requires a bit more qa'ing and hand holding for prod, I think it would be best to have it in a separate PR.